powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/service/behavior/tasks/v3/AuditingServiceBehavior.java

```diff
@@ -51,10 +54,35 @@ public class AuditingServiceBehavior {
     private final SignatureTypeConverter signatureTypeConverter = new SignatureTypeConverter();
     private final KeyValueMapConverter keyValueMapConverter;
 
+    // Generic auditing capability
+    private final Audit audit;
+
     @Autowired
-    public AuditingServiceBehavior(SignatureAuditRepository signatureAuditRepository, KeyValueMapConverter keyValueMapConverter) {
+    public AuditingServiceBehavior(SignatureAuditRepository signatureAuditRepository, KeyValueMapConverter keyValueMapConverter, Audit audit) {
         this.signatureAuditRepository = signatureAuditRepository;
         this.keyValueMapConverter = keyValueMapConverter;
+        this.audit = audit;
+    }
+
+    /**
+     * Log information with specified level, message, audit details, and message args.
+     * @param level Level
+     * @param message Message
+     * @param auditDetail Audit detail
+     * @param args Arguments
+     */
+    public void log(AuditLevel level, String message, AuditDetail auditDetail,  Object... args) {
```

Comment:
What about to have a shortcut `logInfo(String message, AuditDetail auditDetail, Object... args) {audit.log(message, AuditLevel.INFO, auditDetail, args);`